package.json

```diff
@@ -1118,56 +1118,94 @@
-        "title": "Connect to Kafka Clusters",
+        "title": "Get Started with Confluent for VS Code",
         "description": "Interact with Apache Kafka® clusters running locally or on Confluent Cloud.",
         "steps": [
           {
-            "id": "connect-local-cluster",
-            "title": "Connect to Kafka",
-            "description": "**Confluent Cloud Clusters** \nLog in to your Confluent account to see your Confluent Cloud resources. \n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)\n**Local Clusters** \nIf you have Kafka running locally, it appears automatically in the Resources view. \n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "id": "connect-conflulent-cloud",
+            "title": "Connect to Confluent Cloud",
+            "description": "Log in to see your Confluent Cloud resources.\n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
             "media": {
               "image": "resources/walkthrough/connect.png",
               "altText": "Connect to Kafka clusters"
             },
             "completionEvents": [
-              "onCommand:command:confluent.connections.ccloud.signIn",
-              "onCommand:confluent.docker.startLocalResources"
+              "onCommand:command:confluent.connections.ccloud.signIn"
             ]
           },
           {
             "id": "create-connections",
-            "title": "Connect directly to Kafka / Schema Registry",
-            "description": "To connect to an Apache Kafka or Schema Registry cluster, click $(plus) next to Resource > Other. The cluster can be in Confluent Cloud, Confluent Platform, or hosted elsewhere.\n[Open Connection Form](command:confluent.connections.direct)",
+            "title": "Connect Directly",
+            "description": "To connect to a Confluent Platform, WarpStream, Apache Kafka, and/or Schema Registry clusters.\n Select the $(plus) icon next to Resource to create or import a connection.\n[Create Connection](command:confluent.connections.direct)",
             "media": {
               "image": "resources/walkthrough/direct-connection-form.png",
-              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
+              "altText": "Create a new connection to a Kafka cluster and/or Schema Registry"
             },
             "completionEvents": [
               "onCommand:confluent.connections.direct"
             ]
           },
+          {
+            "id": "connect-local-cluster",
+            "title": "Start Local Environment",
+            "description": "If you have Kafka running locally, it appears automatically in the Resources view.\nYou can also start Kafka and Stream Registry clusters using the extension.\n[Start Local Resources](command:confluent.docker.startLocalResources)",
```

Comment:
> If you have Kafka running locally

Hm, I don't know if this is entirely true unless they have local Kafka running through the `confluent-local` Docker image. We should probably keep the second sentence and leave the first one out as an implementation detail.

Reply:
cc @shouples WDYT?

Reply:
IIRC as long as it's running on the default port we will see/show it? I think it's not just implementation details as customers may be running them outside of the extension and want to interact with those via the extension.

Reply:
Yeah, I don't think this will work if someone's running Kafka locally without the `confluent-local` Docker image (exposing Kafka REST on port `8082`).

Reply:
So if they're running the `confluent` CLI it'll get picked up (since it uses that Docker image), but otherwise I wouldn't expect this to work automatically.

Reply:
ACK, updated.